sdcm/tester.py

```diff
@@ -1974,12 +1974,19 @@ def run_fullscan_thread(self, ks_cf='random', interval=1, duration=None):
             interval {number} -- interval between requests in min (default: {1})
             duration {int} -- duration of running thread in min (default: {None})
         """
+        sla_role_name, sla_role_password = None, None
+        if hasattr(self, "fullscan_role"):
+            sla_role_name = self.fullscan_role.name.replace('"', '')
```

Comment:
What all those replaces , can't we do those once when we assign the role parameters ?

Reply:
redundant. remove it